config/initializers/subsites.rb

```diff
@@ -1 +1,34 @@
-SUBSITES = YAML.load_file("#{Rails.root.to_s}/config/subsites.yml")[Rails.env]
\ No newline at end of file
+SUBSITES = YAML.load_file("#{Rails.root.to_s}/config/subsites.yml")[Rails.env].with_indifferent_access
+SUBSITES['sites'] = {}.with_indifferent_access
```

Comment:
Update: Deleting that SUBSITES['sites'] line now in a new commit, as per our Slack conversation.